cli/command/formatter/volume.go

```diff
@@ -45,40 +45,59 @@ func VolumeWrite(ctx Context, volumes []*types.Volume) error {
 		}
 		return nil
 	}
-	return ctx.Write(&volumeContext{}, render)
+	return ctx.Write(newVolumeContext(), render)
+}
+
+type volumeHeaderContext map[string]string
+
+func (c volumeHeaderContext) Label(name string) string {
```

Comment:
@vdemeester I think it might be possible to refactoring. Though it looks tricky. Let me think about it any maybe we could do it in a follow up PR?